pkg/analyzer/analyzer.go

```diff
@@ -88,17 +87,22 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
-func getTargetTypes(pssa *buildssa.SSA, targetPackages []string) []*types.Pointer {
-	targets := []*types.Pointer{}
+func getTargetTypes(pssa *buildssa.SSA, targetPackages []string) []any {
+	targets := []any{}
 
 	for _, sqlPkg := range targetPackages {
 		pkg := pssa.Pkg.Prog.ImportedPackage(sqlPkg)
 		if pkg == nil {
 			// the SQL package being checked isn't imported
-			return targets
```

Comment:
🤔

Reply:
Identical to https://github.com/ryanrolds/sqlclosecheck/pull/24 but `continue` probably makes more sense. 🙂

I will rebase.